src/shared/parse.cpp

```diff
@@ -3547,10 +3545,8 @@ static int Parse_Directive_define( source_t *source )
 	}
 
 	//allocate define
-	define = ( define_t * ) Z_Malloc( sizeof( define_t ) + strlen( token.string ) + 1 );
-	Com_Memset( define, 0, sizeof( define_t ) );
-	define->name = ( char * ) define + sizeof( define_t );
-	strcpy( define->name, token.string );
+	define = (define_t *) calloc( sizeof( define_t ), 1 );
```

Comment:
The allocation function probably shouldn't be changed, since I don't see any corresponding change to a freeing function.